reana_client/cli/workflow.py

```diff
@@ -81,7 +81,14 @@ def workflow_execution_group(ctx):
     help="Get output in JSON format.",
 )
 @click.option(
-    "--all", "show_all", count=True, help="Show all workflows including deleted ones."
+    "--all", "show_all", count=True, help="Show all workflows including deleted ones.",
+)
+@click.option(
```

Comment:
As discussed IRL, since we'll introduce filters in `master`, we can rely here on just flipping the default, so that CLI UI and Web UI would return the same thing on 0.7, and if somebody would like to filter deleted ones, they would be able to do something like:

```console
$ reana-client list | grep -v 'deleted$'
```

Please keep `--all` for backwards compatibility even if the default is true...